app/javascript/hyacinth_ui_v1/components/users/UserEdit.js

```diff
@@ -97,29 +93,33 @@ function UserEdit() {
     if (manageVocabularies) permissions.push('manage_vocabularies');
     if (readAllDigitalObjects) permissions.push('read_all_digital_objects');
     if (manageAllDigitalObjects) permissions.push('manage_all_digital_objects');
-
-    return updateUser({
-      variables: {
-        input: {
-          id,
-          firstName,
-          lastName,
-          email,
-          isActive,
-          isAdmin,
-          currentPassword,
-          password,
-          passwordConfirmation,
-          permissions,
-        },
+    const userData = {
+      input: {
+        id,
+        firstName,
+        lastName,
+        email,
+        currentPassword,
+        password,
+        passwordConfirmation,
       },
+    };
+    if (ability.can('manage', 'User')) {
+      if (!isAdmin) userData.input.isActive = isActive;
+    }
+    if (authenticatedUser.isAdmin) {
```

Comment:
This can be converted to `ability.can('manage', 'all')`.

Reply:
hmm, I didn't see that in the permissions I inspected - it was `('manage', 'Users')`, which is definitely a mistake. Is all what we want?